### ISSUE TITLE
eager is not compatible with Nextflow 21.04

### ISSUE THREAD

**Issue by jfy133** [nf-core/eager#741]
The following error is found on start up 

```
:dna: 21:10:48 with 6GiB/15GiB in eager/tests on  bcftools [!] ✖  nextflow run nf-core/eager -r 2.3.4 -profile conda,test_tsv --run_genotyping --genotyping_tool 'ug' --run_bcftools_stats --run_multivcfanalyzer --run_vcf2genome -with-tower
N E X T F L O W  ~  version 21.04.0
Launching `nf-core/eager` [trusting_stonebraker] - revision: 10bfbdff86 [2.3.4]
Unknown method invocation `isEmpty` on Boolean type
```

Possibly related to: https://github.com/nf-core/tools/issues/992#issuecomment-826707802

Should change all empty strings to `null`, and check validations still work.

**Comment by jfy133** [comment]
Ok actually seems to be with the `-profile conda`, but should do clean up anyway

**Comment by jfy133** [comment]
This was a bug (I made ) in the template . Should be fixed in the template merge of tools 1.14
